src/utils/i18n.ts

```
@@ -20,12 +26,18 @@
-
+  // Configuração opcional para preferência de línguas
+  load: 'languageOnly',
+  // Ordem de preferência para línguas
+  supportedLngs: languages,
   interpolation: {
     escapeValue: false,
   },
 });
-export default i18next;
+
+export default i18next;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)